homeassistant/components/fyta/sensor.py

```diff
@@ -33,9 +33,7 @@
 class FytaSensorEntityDescription(SensorEntityDescription):
     """Describes Fyta sensor entity."""
 
-    value_fn: Callable[[str | int | float | datetime], str | int | float | datetime] = (
-        lambda value: value
-    )
+    value_fn: Callable[[Plant], str | int | float | datetime | None]
```

Comment:
```suggestion
    value_fn: Callable[[Plant], StateType | datetime]
```

Reply:
Not sure if this is correct. I added all values to Plant and therefore respective value_fn for all sensors, so that (according to my understanding) str, int or float values can be returned.

Reply:
StateType == `type StateType = str | int | float | None` so this looks cleaner :)